client/src/api/connectJSON.js

```diff
@@ -6,7 +6,9 @@ export async function getRecipies() {
 }
 
 export async function getRecipeByRecipeName(RecipeName) {
-  const response = await fetch(`http://localhost:8080/recipes/${RecipeName}`);
+  const response = await fetch(
+    `http://localhost:8080/recipes?RecipeName=${RecipeName}`
+  );
```

Comment:
You could think of using a proxy, so you could shorten this
```
const response = await fetch(
    `/recipes?RecipeName=${RecipeName}`
);
```

Reply:
thanks!!!:) :)
I'm right on it with the proxy! :)